print/components/PicassoChunk.vue

```diff
@@ -85,27 +101,33 @@ export default {
 
       return `[${responsibles}]`
     },
+
+    dayResponsiblesCommaSeparated(date) {
+      const day = this.getDayByDate(date)
+      if (!day) return null
+      return dayResponsiblesCommaSeparated(day, this.$tc.bind(this))
+    },
```

Comment:
You have added this function once in this component and once in print/helpers, but really it should probably be in common and have unit tests, don't you think?

Reply:
Yeah, probably makes sense to put this in commons and we could then potentially use if here as well:
https://github.com/ecamp/ecamp3/pull/3022/files#diff-fcb428166931633c8e0d0a8df3f6398f533d8406a91d437df0de24a83b0fe8b6R12

Reply:
Move in commit https://github.com/ecamp/ecamp3/pull/3026/commits/50484442f0499ead958511a32abb2a1e962c55c4